pkgs/servers/squid/default.nix

```diff
@@ -0,0 +1,27 @@
+{ fetchurl, stdenv, perl, lib
+, openldap, pam, db, cyrus_sasl, libcap, expat, libxml2, libtool
+, openssl, ... }:
+{
+  squid35 = stdenv.mkDerivation rec {
```

Comment:
The attribute set wrapping the `mkDerivation` call must be removed because `callPackage` is used in `all-packages.nix`. This causes the Travis build error.